presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -295,6 +295,8 @@ literal
     | booleanValue                                                                   #booleanLiteral
     | STRING                                                                         #stringLiteral
     | BINARY_LITERAL                                                                 #binaryLiteral
+    | ARRAY '[' (literal (',' literal)*)? ']'                                  #arrayLiteralConstructor
```

Comment:
Why not allow full expressions for parsing `EXECUTE`, but fail when replacing if the expression contains an identifier? This is similar to `CALL` or `SET SESSION`.